source/deployment/mobile-app-deployment.rst

```diff
@@ -0,0 +1,577 @@
+- Do you know what devices will be used if BYOD?
+- What OS do you want to start with in testing?
+- Creating a sample configuration then run validation tests against each config item
+
+**********************************
+Configuring Mattermost to Use HPNS
+**********************************
+
+Configuring your Mattermost Server to use the Mattermost HPNS is a single configuration item. This is covered in our `Hosted Push Notification Service documentation <https://docs.mattermost.com/mobile/mobile-hpns.html>`_. 
+
+Next your users would need to install the mobile application on their device. If desired, you can further configure security capabilities using an EMM provider.
+
+**********************************
+Updating via the Public Store Apps
+**********************************
+
+While not part of your initial mobile deployment, you should consider a strategy for updating when new versions of the Mattermost mobile applications are available. Simultaneously, you should check any compatibility requirements for the mobile apps and the Mattermost server. 
+
+For example we recommend you:
+- Check the compatibility requirements
+- Validate versions connecting to server
+- Update server
+- Update apps
+
+It is often easier to upgrade the mobile apps. However, not all provided updates are compatible with all previous versions of the Mattermost server.
+
+Consult the `Mattermost mobile app changelog <https://github.com/mattermost/mattermost-mobile/blob/master/CHANGELOG.md>`_ and `Mattermsot server changelog <https://docs.mattermost.com/administration/changelog.html>`_ for more information.
+
+    **Note:** Only updating the mobile apps, or updating the mobile apps before the Mattermost server, can result in incompatibility.
+
+----
+
+###################################
+Mobile Deployment via Custom Builds
+###################################
+
+Choosing this model means you’ve decided not to use the mobile applications Mattermost has made available via the public app stores. 
+
+This also means you will need to maintain these applications. Maintenance includes rebuilding and incorporating feature and/or security updates. Otherwise, your applications will not match the functionality of our publicly available applications, and could be incompatible with future versions of the Mattermost server.
+
+Finally, this choice also means you cannot use the hosted version of the Mattermost Push Notification Service (HPNS).
+
+A great first step for this deployment model involves `reviewing the documentation <https://developers.mattermost.com/contribute/mobile/>`_. Your development team should run through the requirements for building the applications. This will help understand the overall impact when choosing this option. This documentation provides guidance on building, compiling, and signing. It also includes information for customizing the apps.
+
+We’ve provided some recommended sections below.
+
+- Developer Setup
```

Comment:
These could have links 0/5

Reply:
has links in the dedicated sections directly below, should be enough i hope ;)